<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
    <link rel="stylesheet" href="style.css">
    <title>Dzianis Prakapenka</title>
</head>
<body>
    <div class="grid-2">
        <div class="section-1">
            <i class="fas fa-code fa-5x white"></i>
            <h2>Dzianis Prakapenka</h2>
            <p>Minneapolis, USA.</p>
            <!--<a href="#"><i class="fab fa-twitter"></i></a>-->
            <a href="https://www.linkedin.com/in/dzianis-prakapenka/"><i class="fab fa-linkedin"></i></a>
            <!--<a href="#"><i class="fab fa-github"></i></a>-->
        </div>
        <div class="section-2">
            <h2>About</h2>
            <p>I am an experienced software developer and computational biologist with a strong background in developing and optimizing computing pipelines, tools, and methods for working with large datasets. My research focus has primarily revolved around genomic prediction, haplotype analysis, and quantitative trait estimation in various species, including cattle, pigs, wheat, and human phenotypes. I am well versed in Python, C++, Fortran, and R (specifically for high performance scientific computing tasks) and live in Linux.</p>

            <h2>Experience</h2>
	    <p>As a Postdoctoral Associate at the University of Minnesota, I wrote software to support integration of complex effects in genomic prediction software and conducted genomic analyses of large datasets (samples of over one million individuals).  I hold a Ph.D. in Bioinformatics and Computational Biology, where one of the central contributions of my thesis was building software to incorporate haplotypes in genomic prediction. This involved developing, implementing, and testing a pipeline that helps with complex data preparation required for haplotype analysis and designing and writing software that dynamically supports the use of large datasets on computing clusters with customized optimizations for scalability, load balancing, and resource utilization.</p>
	<p>Prior to my Ph.D., I developed scientific computing and visualization software to support research across various fields at the Research Computing Center at the University of Chicago and at Oak Ridge National Laboratory.</p>

            <h2>Skills</h2>
            <p>Python, C/C++, Fortran MKL | LAPACK, ScaLAPACK, OpenMP/MPI | R, Matlab, Mathematica, Stata | Java, Visual Basic, Basic, IDL | Linux, OS X, Windows, Chrome OS | Bash, Slurm Workload Manager, Portable Batch System</p>
            <!-- <h2>Projects</h2>
            <a href="#">Project 1</a>
            <a href="#">Project 2</a>
            <a href="#">Project 3</a>
            <a href="#">Project 4</a>
            <a href="#">Project 5</a> -->
            <h2>Contact</h2>
            <p>praka032_at_umn_dot_edu</p>
        </div>
    </div>
    &nbsp; &nbsp;
    <font size="1">
	    <center>
		    Website source code from <a href="https://github.com/giotsere/minimalist-portfolio">here</a>.
	    </center>
    </font>
</body>
</html>
